### PR TITLE
update omjulia doc

### PR DESCRIPTION
**Pull request by arun3688**

### CODE REVIEW

File: UsersGuide/source/omjulia.rst [hunk 1]
```diff
@@ -19,55 +19,228 @@ ZeroMQ - high performance asynchronous messaging library and it supports the Mod
...
 - Able to interact with the OpenModelica compiler through the :ref:`available API <scripting-api>`
+- Easy access to the Modelica Standard library.
 - All the API calls are communicated with the help of the sendExpression method implemented in a Julia module
 - The results are returned as strings
 
 Test Commands
 ~~~~~~~~~~~~~
 
-To get started, create a OMJulia session object:
+To get started, create a omc OMJulia session object:
```
Comment (sjoelund) [inline UsersGuide/source/omjulia.rst]: a -> an

Also, why refer to omc? It's an OMJulia object

File: UsersGuide/source/omjulia.rst [hunk 2]
```diff
@@ -19,55 +19,228 @@ ZeroMQ - high performance asynchronous messaging library and it supports the Mod
...
 
 >>> using OMJulia
->>> omc= OMJulia()
+>>> omc= OMJulia.OMCSession()
 >>> omc.sendExpression("getVersion()")
-'v1.13.0-dev-531-gde26b558a (64-bit)'
+"v1.13.0-dev-531-gde26b558a (64-bit)"
```
Comment (sjoelund) [inline UsersGuide/source/omjulia.rst]: Shouldn't this ask OMPython for the version? We shouldn't have other versions hardcoded in the docs

**Comment by sjoelund** [comment]
Also, sign the CLA

File: UsersGuide/source/omjulia.rst (hunk 2 above)
Comment (arun3688) [reply UsersGuide/source/omjulia.rst]: This is OMJulia, i think the docs does not use julia to compute anything, May be i can remove the getVersion()

**Comment by arun3688** [comment]
Fixed the comments and signed the CLA

**Pull request merged**